code/frameworks/solidjs-vite/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@storybook/solid-vite",
```

Comment:
Go to `code/lib/cli/src/versions.ts` and add `@storybook/solid-vite` there as well

Reply:
👍